.env.example

```diff
@@ -1,9 +1,17 @@
 PORT=<PORT_NUMBER>
 DB_CONNECT = <DATABASE URL>
 TEST_DB=<TEST DATABASE URL>
-LOCAL_HOST=http://localhost:${PORT}
 FB_CLIENT_ID: <FB CLIENT ID>
 FB_CLIENT_SECRETE: <FB CLIENT SECRETE>
 GOOGLE_CLIENT_ID: <google CLIENT ID>
 GOOGLE_CLIENT_SEC: <GOOGLE CLIENT SEC>
 TOKEN_EXPIRE:<TOKEN EXPIRE>
+HOST=http://localhost:${PORT}
```

Comment:
will this work for a host that is not LocalHost?

Reply:
I missed this one, it should not be precise to localhost. Depending on the environment the value differ to adapt to the host of said environment